content/md/productivity/get-started-with-the-rules-engine.md

```diff
@@ -253,6 +254,40 @@ To concatenate the **brand** (non localizable and non scopable) and the **model*
           field: product_title
           locale: en_US
 ```
+## Clear
+This action clears the value(s) assigned to an attribute, product categories, product groups, or product associations.
+
+One parameter is required while the two others are optional. The expected values are:
+ - `field`: the attribute code, "categories", "groups" or "associations".
+ - `locale`: the locale code for which the value is assigned. For localizable attributes (optional).
+ - `scope`: the channel code for which the value is assigned. For scopable attributes (optional).
+
```

Comment:
When I read this part, I don't understand what is required and what is optional.